pkgs/tools/security/genpass/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, git
+, openssl
+, pkgconfig
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "genpass";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "cyplo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1b22m7g55k5ry0vwyd8pakh8rmfkhk37qy5r74cn3n5pv3fcwini";
+  };
+
+  cargoSha256 = "1p6l64s9smhwka8bh3pamqimamxziad859i62nrmxzqc49nq5s7m";
+
+  buildInputs = [
+    git
+    openssl
+    pkgconfig
+  ];
+
```

Comment:
```suggestion
```

I was able to build without these. (Also should be removed from the top of the file.)

Reply:
ah yes, good catch !